02LessonTwo/homework/css/style.css

```diff
@@ -0,0 +1,108 @@
+html {
+    background-image: url("https://wallpaperscraft.com/image/paper_crumpled_light_shadow_35117_1920x1080.jpg");
```

Comment:
I like this! Great job, you are showing me you have creativity and that is what it takes to become a top notch web developer.